pkg/controller/acmechallenges/controller.go

```diff
@@ -135,7 +140,34 @@ func (c *controller) Register(ctx *controllerpkg.Context) (workqueue.TypedRateLi
 	}
 
 	// register handler functions
-	if _, err := challengeInformer.Informer().AddEventHandler(&controllerpkg.QueuingEventHandler{Queue: c.queue}); err != nil {
+	if _, err := challengeInformer.Informer().AddEventHandler(&controllerpkg.QueuingEventHandler{
+		Queue: c.queue,
+		Predicates: []predicate.TypedPredicate[client.Object]{
```

Comment:
@wallrj-cyberark maybe the queue and predicates can use the *cmacme.Challenge type instead of client.Object?
We can do the casting in the event handler.